x-pack/test/functional/apps/remote_clusters/ccs/remote_clusters_index_management_flow.ts

```diff
@@ -88,8 +89,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
       it('Verify that the follower index is duplicating from the remote.', async () => {
         await pageObjects.indexManagement.clickIndexAt(0);
-        await pageObjects.indexManagement.performIndexAction('flush');
-        await testSubjects.click('indexDetailsBackToIndicesButton');
```

Comment:
Just for my own understanding, why are these actions not needed anymore?

Reply:
New searchIndex page has not flush function anymore :(

Reply:
we shouldn't be updating these tests.

These should still be using the index management details page. But that will likely be handled by @saarikabhasi  PR.

Reply:
Returned this changes